dlp/api/DlpSample/RedactSamples.cs

```diff
@@ -1,4 +1,18 @@
-﻿using Google.Api.Gax.ResourceNames;
+﻿// Copyright 2018 Google Inc.
```

Comment:
Was this file actually created in 2018? (Fine if so, obviously.)

Reply:
Yep, March 2018 :).